src/utils.js

```diff
@@ -0,0 +1,163 @@
+    label.style['background-color'] = '#fff';
+  });
+}
+
+function scrapeAndSave() {
+  const tasksArray = [];
+  const tasks = document.querySelectorAll('.task');
+  tasks.forEach((label, index) => {
+    const task = new Task(label.innerHTML, false, index + 1);
+    tasksArray.push(task);
+  });
+  localStorage.removeItem('tasksList');
+  localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+}
+
+function setValueTextArea(textArea, labelTask) {
+  labelTask.innerHTML = textArea.value;
+  textArea.style.display = 'none';
+  labelTask.style.display = 'block';
+  // restore original icons
+  restoreOriginalIcons();
+  // change background to white
+  restoreBackgroundWhite();
+  // Save to local storage
+  scrapeAndSave();
+}
+
+export function addEventclearAllButton() {
+  const buttonClearAll = document.querySelector('.clearAll');
+  buttonClearAll.addEventListener('click', () => {
+    const liElements = document.querySelectorAll('li');
+    liElements.forEach((liElem) => {
+      const checkboxElem = liElem.querySelector('.checkBoxesTasks');
+      if (checkboxElem.checked) {
+        liElem.remove();
+      }
+    });
+    scrapeAndSave();
+  });
+}
+
+export function manageBoxes(checkboxElem) {
+  const parentDiv = checkboxElem.parentNode;
+  const textChild = parentDiv.querySelector('.task');
+  if (checkboxElem.checked) {
+    const strike = textChild.getElementsByTagName('strike')[0];
+    if (strike) {
+      textChild.innerHTML = strike.innerHTML;
+    }
+    textChild.innerHTML = textChild.innerHTML.strike();
+  } else {
+    const strike = textChild.getElementsByTagName('strike')[0].innerHTML;
+    textChild.innerHTML = strike;
+    // restore visual
+    const textArea = parentDiv.querySelector('.textArea');
+    textArea.style.display = 'none';
+    const labelText = parentDiv.querySelector('.task');
+    labelText.style.display = 'block';
+    // restore original icons
+    restoreOriginalIcons();
+    // change background to white
+    restoreBackgroundWhite();
+  }
+}
+
+export function addEventsCheckBoxes() {
+  const checkBoxes = document.querySelectorAll('.checkBoxesTasks');
+  checkBoxes.forEach((box) => {
+    box.addEventListener('change', () => { manageBoxes(box); });
+  });
+}
+
+function restoreDataTExtArea(lilElem, labelTask, textArea) {
+  lilElem.style['background-color'] = '#fffee4';
+  labelTask.style.display = 'none';
+  textArea.style.display = 'block';
+  textArea.value = labelTask.innerHTML;
+  // change incon to trash
+  const iconDiv = lilElem.querySelector('.icon');
+  iconDiv.innerHTML = '&#x1F5D1';
+  textArea.focus();
+  textArea.addEventListener('blur', () => { setValueTextArea(textArea, labelTask); });
+  textArea.addEventListener('keypress', (event) => {
+    if (event.keyCode === 13) {
+      if (textArea.value !== '') {
+        setValueTextArea(textArea, labelTask);
+      }
+    }
+  });
+}
+
+export function manageLabels(labelTask) {
+  const parentDivTaskCont = labelTask.parentNode;
```

Comment:
- [ ] Hi @alex1779, Well done working on your Todo list project by following the sneak peek and project requirements as much as possible, but I would like to raise a suggestion about what this particular project milestone is all about. According to the sneak peek and requirement, you are to add and delete a list which I can see the adding list is working perfectly well. But I would advise you to implement the delete functionality on each list whenever you try to edit a delete icon pops up kindly implement the functionality on the icon. This will make us be able to delete a particular list out the all the lists we have.